feature-libs/order/README.md

```diff
@@ -2,4 +2,8 @@
 
 Spartacus Order feature can be added to the existing Spartacus application by running `ng add @spartacus/order`. For more information about Spartacus schematics, visit the [official Spartacus schematics documentation page](https://sap.github.io/spartacus-docs/schematics/).
 
+To see enhanced UI in Order History and Order Details page in B2C storefront, 
+1. Add `{ provide: MYACCOUNT_V2_ORDER, useValue: true },` to order-feature.module.ts file `providers` array.
```

Comment:
[MEDIUM]
For consistency, let's treat "my account" as 2 words `MY_ACCOUNT_V2_ORDER`.
It's part of our public API, so should be fixed before the release.

Note: an analogical remark applies also to other places in this PR, considering various case encodings:
`myaccount-kebab` -> `my-account-kebab`
`MYACCOUNT_CONSTANT` -> `MY_ACCOUNT_CONSTANT`
`myaccountCamel` -> `myAccountCamel`
`MyaccountPascal` -> `MyAccountPascal`